test/Microsoft.AspNet.Mvc.Core.Test/Controllers/DefaultControllerFactoryTest.cs

```diff
@@ -257,7 +215,12 @@ private IServiceProvider GetServices()
 
         private static DefaultControllerFactory CreateControllerFactory(IControllerActivator controllerActivator = null)
         {
-            controllerActivator = controllerActivator ?? Mock.Of<IControllerActivator>();
+            var controllerActivatorMock = new Mock<IControllerActivator>();
+            controllerActivatorMock
```

Comment:
Then the verification needs to be if `controllerActivatorMock.Release` is invoked. You shouldn't need to dispose it.